topics/gatsby/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 aliases: gatsbyjs, gatsby-js
-created_by: Ilesh Mistry
+created_by: Kyle Mathews
```

Comment:
Should we list both Kyle Mathews and Sam Bhagwat here? According to gatsbyjs.com/about/ they are both co-founders.
```suggestion
created_by: Kyle Mathews & Sam Bhagwat
```